CHANGELOG.md

```diff
@@ -1,3 +1,55 @@
+## 0.9.2
```

Comment:
How come this isn't going to 0.10.0 for the breaking change?

Reply:
Good point!